test_images.py

```diff
@@ -0,0 +1,68 @@
+import wake_vision_loader
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import data_filters
+from experiment_config import get_cfg
+
+cfg = get_cfg()
+
+ds = tfds.load(
+    "partial_open_images_v7",
+    data_dir=cfg.WV_DIR,
+    shuffle_files=False,
+)
+
+
+test = wake_vision_loader.open_images_to_wv(ds["test"], "test", cfg)
+
+test_person = test.filter(data_filters.person_filter)
+test_non_person = test.filter(data_filters.non_person_filter)
+
+
+test_person_bright = data_filters.get_high_lighting(test_person)
+test_person_normal_light = data_filters.get_medium_lighting(test_person)
+test_person_dim = data_filters.get_low_lighting(test_person)
+test_non_person_bright = data_filters.get_high_lighting(test_non_person)
+test_non_person_normal_light = data_filters.get_medium_lighting(test_non_person)
+test_non_person_dim = data_filters.get_low_lighting(test_non_person)
+test_far_persons = test.filter(
```

Comment:
This needs to be test_person.filter to ensure we don't have depictions in there.

Also, I've updated the ranges in this PR https://github.com/Ekhao/Wake_Vision/pull/25

Reply:
This is in the deleted file, so it shouldn't matter!